.github/workflows/config-update.yml

```diff
@@ -0,0 +1,68 @@
+name: Update Deployment Config
+
+on:
+  workflow_dispatch:
+    inputs:
+      deployment:
+        required: true
+        type: string
+        description: Name of the deployment to update config for (e.g. nrel-commute)
+      script_name:
+        required: true
+        type: choice
+        options:
+          - update_admin_access
+      script_args:
+        required: false
+        type: string
+        description: Additional arguments for the script (space-separated)
+      token:
+        required: false
+        type: string
+        description: GitHub token for authentication. If not provided, the workflow will use the default token.
+
+env:
+  GITHUB_TOKEN: ${{ inputs.token || secrets.GITHUB_TOKEN }}
```

Comment:
will the `GITHUB_TOKEN` really work?
- If so, we don't need `inputs.token`
- If not, we can just fail early

Reply:
We need `inputs.token` because PRs created by the @github-actions bot (ie the default GITHUB_TOKEN) don't trigger checks https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#triggering-further-workflow-runs, so merging would be blocked without us manually running the checks

By passing in `inputs.token`, the PR can be created by @op-config-updates, which is a bot I created

Reply:
If we need `inputs.token`, then why have `inputs.token || secrets.GITHUB_TOKEN`?
can't you just remove the part after the `||` and fail early?